runtime/centrifuge/Cargo.toml

```diff
@@ -127,6 +131,11 @@ pallet-restricted-tokens = { path = "../../pallets/restricted-tokens", default-f
 pallet-rewards = { path = "../../pallets/rewards", default-features = false }
 runtime-common = { path = "../common", default-features = false }
 
+# LiquidityPools 3rd-party dependencies
+moonbeam-relay-encoder = { git = "https://github.com/PureStake/moonbeam", default-features = false, rev = "00b3e3d97806e889b02e1bcb4b69e65433dd805d" }
```

Comment:
why do we need that?

Reply:
For the HRMP encoder here - https://github.com/centrifuge/centrifuge-chain/blob/feat/add-lp-to-centrifuge-runtime/runtime/centrifuge/src/lib.rs#L1298, which we might also have to change and use the Polkadot on as William suggested.